### DISCUSSION TITLE
Wifi not connecting after power outage

### DISCUSSION THREAD

**Discussion by rolfhans** [arendst/Tasmota#11556] [Q&A] [answered]
Hi, after poweroutage an ESP32 with Tasmota cannot connect to WiFi. There is a Fritz Repeater needing a long time to start up. Tasmota seem to stop connecting to wifi till then. Powercycling the ESP32 brings it into Wifi. Testing keeping the ESP32 with power and just switch off the repeater and on again after a time Tasmota is connecting. Just after the outage of both and coming back to power at the same time isn't working. I tried Wificonfig 4 and 5. I entered one accesspoint and the same accesspoint as a second access point. I tried setOption 56, nothing working. I really think, after a power outage, a Home automation should come up usable. I am new at this and might overlook something. What I was searching for is an option to set the time, Tasmota is searching for wifi at startup befor doing anything else. 
Any Idea?
Rolf

**Comment by barbudor** [comment, marked as answer]
Hi, with `WifiConfig 5` Tasmota should reconnect as soon as the AP is available. No need for defining twice the same SSID (unless you have multiple AP and want Tasmota to be able to switch to the other AP)

Is that an ESP32 dev board ? Can you hook a serial terminal (Termite, Putty, minicom, ...) and check what is happenning when Tasmota starts with AP not reachable ?

For your information, with power outage there is a risk of non-clean restart when power doesn't come back straight but show multiple on/off cycle. This could trigger one of the recovery process that erase Tasmota configuration and restart AP mode.
This was not your case, but still important that you have an understanding of that [Recovery](https://tasmota.github.io/docs/Device-Recovery/) doc section.

**Comment by rolfhans** [comment]
Thanks, I’ll try later to connect a serial monitor from the Arduino IDE. After description it was my understanding it should reconnect with wificonfig 5, it is just not doing it.
It is an ESP32 development board from AZ Delivery. Something with wroom printed on it.
Rolf

Von meinem iPhone gesendet

Am 02.04.2021 um 14:49 schrieb Barbudor ***@***.***>:

﻿

Hi, with WifiConfig 5 Tasmota should reconnect as soon as the AP is available. No need for defining twice the same SSID (unless you have multiple AP and want Tasmota to be able to switch to the other AP)

Is that an ESP32 dev board ? Can you hook a serial terminal (Termite, Putty, minicom, ...) and check what is happenning when Tasmota starts with AP not reachable ?

For your information, with power outage there is a risk of non-clean restart when power doesn't come back straight but show multiple on/off cycle. This could trigger one of the recovery process that erase Tasmota configuration and restart AP mode.
This was not your case, but still important that you have an understanding of that Recovery<https://tasmota.github.io/docs/Device-Recovery/> doc section.

**Reply by rolfhans** [reply]
Hi, back from the cellar. I don't know from my side how to investigate that deeper and would give up. Trying to connect the wifi repeater as simultane as possible to power while the ESP32 development board to the PC for getting monitor data, it always does, what is expected. After several tries it connects to the access point as soon as wifi is available. Connecting both, the repeater and the power supply of the ESP32 to the same outlet and switching it on simultaneously: no connection. I don't know, how to get monitor data in this situation from the ESP32, sorry.

**Reply by Jason2866** [reply]
Repeaters are known troublemakers in combination with ESP82XX and ESP32.
Until now we have no feedback from users which have solved this issue.

**Reply by rolfhans** [reply]
In my case, it points more to the power adapter which is a normal Micro-USB charger. I need to find a different one. The guess is because connecting the ESP32 to the Laptop to get the serial monitor worked several times without problem. I might not have switched on both, the repeater and the ESP in the same millisecond but the repeater was still booting up. The Tasmota on the ESP32 Dev Board did what it should do. And yes, it did it also be powercycling it with the current power supply. Nevertheless seeing the serial monitor output, I am 100% confident, that Tasmota does it's job correct. Let me play arround a little more. I'll give information if I find out more.
